catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -564,16 +566,15 @@ public Instant parseToDate(Serializable value) {
   private Pattern boonDefault =
       Pattern.compile("[a-zA-Z]{3}\\s[a-zA-Z]{3}\\s\\d+\\s[0-9:]+\\s(\\w+\\s)?\\d+");
 
-  private Pattern iso8601 =
-      Pattern.compile("\\d+-?\\d+-?\\d+T\\d+:?\\d+:?\\d+(Z|([+\\-])\\d+:\\d+)");
+  private Pattern iso8601Z = Pattern.compile("\\d+-?\\d+-?\\d+T\\d+:?\\d+:?\\d+(\\.\\d+)?Z");
+
+  private Pattern iso8601Offset =
+      Pattern.compile("\\d+-?\\d+-?\\d+T\\d+:?\\d+:?\\d+(\\.\\d+)?[+\\-]\\d+:\\d+");
 
   public Serializable parseDate(Serializable value) {
```

Comment:
❓ If we're consistently returning an Instant, should we have this method return that instead of Serializable?

Reply:
Thought about that, was a little afraid of changing the signature of the method, but I see it is not whitelisted